scripts/update-sdk-version.js

```diff
@@ -0,0 +1,65 @@
+const fs = require('fs');
+const path = require('path');
+// const process = require('process');
+// const exec = require('child_process').spawnSync;
+const { logger, spawnOrFail, process } = require('./utilities');
+
+// const spawnOrFail = (command, args, options) => {
```

Comment:
nit: Need to remove all this comments.

Reply:
Sure, it is just for testing, not the final version :)